### PR TITLE
Add Projects Creation Button to Projects Page

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4543

projects-button

### CODE REVIEW

File: pages/contribute.js [hunk 1]
```diff
@@ -273,10 +273,15 @@ class TiersPage extends React.Component {
...
-                        <Link href={`/${collective.slug}/events/new`}>
+                      {LoggedInUser?.canEditCollective(collective) && (verb === 'events' || verb === 'projects') && (
+                        <Link href={`/${collective.slug}/${verb}/new`}>
                           <StyledButton buttonStyle="primary">
-                            <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
+                            {verb === 'events' && (
+                              <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
+                            )}
+                            {verb === 'projects' && (
+                              <FormattedMessage id="project.create.btn" defaultMessage="Create Project" />
```
Comment (znarf) [inline pages/contribute.js:283]: We already have a `SectionProjects.CreateProject` string? Could you check for duplicates when introducing new strings?

Comment (SudharakaP) [reply pages/contribute.js]: Ops, didn't thought about that. Will do so in the future. I've corrected this. 👍🏽

File: pages/contribute.js [hunk 2]
```diff
@@ -273,10 +273,15 @@ class TiersPage extends React.Component {
...
+                      {LoggedInUser?.canEditCollective(collective) && (verb === 'events' || verb === 'projects') && (
+                        <Link href={`/${collective.slug}/${verb}/new`}>
                           <StyledButton buttonStyle="primary">
-                            <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
+                            {verb === 'events' && (
+                              <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
+                            )}
+                            {verb === 'projects' && (
+                              <FormattedMessage id="SectionProjects.CreateProject" defaultMessage="Create Project" />
+                            )}
```
Comment (kewitz) [inline pages/contribute.js:284]: I think just duplicating the event's code would look simpler but this also works.

**Review by kewitz — APPROVED** [review]

File: pages/contribute.js (hunk 2 above)
Comment (SudharakaP) [reply pages/contribute.js]: Ah, I thought about this initially and then thought the other approach has less duplicated lines. Anyways, I am okay with this as well and therefore changed the code to this; it definitely looks simpler. 👍🏽

**Pull request merged**